qa/L0_perf_nomodel/test.sh

```diff
@@ -161,6 +187,13 @@ for idx in "${!TEST_NAMES[@]}"; do
     TEST_INSTANCE_COUNT=${TEST_INSTANCE_COUNTS[$idx]}
     TEST_CONCURRENCY=${TEST_CONCURRENCY[$idx]}
 
+    # FIXME: If PA C API adds SHMEM support, remove this.
+    if [[ "${BENCHMARK_TEST_SHARED_MEMORY}" != "none" ]] && \
+       [[ "${TEST_PROTOCOL}" == "triton_c_api" ]]; then
+      echo "WARNING: Perf Analyzer C API doesn't currently support shared memory, skipping."
```

Comment:
Perf Analyzer does not support shared memory I/O when benchmarking directly with Triton C API, skipping.